### PR TITLE
w01d05

### PR DESCRIPTION
**Pull request by annieredmond1**
comfort: 4
completeness: 5

I got off to a decent start but then ended up getting frustrated and having to start all the way over because I realized putting the click listeners inside functions wasn't working.  It still took a long time but felt really good when I finished.  This is my second time doing the pull request (I didn't read the instructions the first time) so I hope it works!

### CODE REVIEW

**Comment by annieredmond1** [comment]
w02d01

comfort 4 
complete 4

I really wish we had read the homework before trying the closure exercises - it would have definitely saved some frustration.  I feel ok with the reading, but would definitely like to do some examples in class.

**Comment by annieredmond1** [comment]
w02d02

comfort: 5
complete: 5

The reading was clear and helpful - I really like the Zapier reading.

**Comment by annieredmond1** [comment]
w02d03

comfort: 5
complete: 5

I did the homework before the new one was posted, but I think it ended up the same :)

**Comment by tgaff** [comment]
Hi @annieredmond1 thanks for putting your comments everyday.  :-) 

I see in the commit log that the commits came from "YOUR-USERNAME".  (you can see that above under your first comment).  If your git is asking your to set a global email and username, you should do so.  There are also instructions about that in the installfest doc.

File: annieredmond1/TicTacToe/ticTacToe.js [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+			}
+			count++;
+		} else {
+			alert("This sqaure already taken, please choose another!");
+		}
+		getWinner();
+		whoseTurn();
+	});
+
+	$('#five').on('click', function () {
```
Comment (tgaff) [inline annieredmond1/TicTacToe/ticTacToe.js]: Each of these functions is VERY similar.  Can you think of a way to only write this function once and still have it work?  (DRY)

**Pull request merged**